Adding missing icon assets

niels9001 opened the pull request:
This PR:

- Adds a few icons that were missing in the docs folder
- Minor updates to the `Peek` icon (agreed upon with @SamChaps)
- Adding .ico's for all WinUI 3 applications that require it, supporting all file sizes: 

![image](https://github.com/microsoft/PowerToys/assets/9866362/add73275-2a48-4ae3-ab2b-d33174a3c4f3)

niels9001 committed:
File Locksmith

niels9001 committed:
Replacing more assets

Jay-o-Way commented:
NIT: 
![image](https://github.com/microsoft/PowerToys/assets/65828559/aca69aa2-91fd-4f4f-9e03-9e498f07c244)
(H=25?)

Jay-o-Way commented:
I see **40px** but not **48px**, is that right? I think there should be 48.
And (looking at [this list](https://learn.microsoft.com/nl-nl/windows/apps/design/style/iconography/app-icon-construction#app-icon)) these `AppList.Scale-***.png` files are deemed "optional", where a couple of others are labeled "required". Don't know if that matters.

jaimecbernardo commented:
This seems to work well from what I've tested, everything still runs.
@Jay-o-Way makes a valid point, though. The dimensions seem to be off for some of the assets, which can affect the way they'll be displayed?

niels9001 committed:
Fixing image dimensions

niels9001 commented:
> This seems to work well from what I've tested, everything still runs. @Jay-o-Way makes a valid point, though. The dimensions seem to be off for some of the assets, which can affect the way they'll be displayed?

Replaced the images that were having a weird offset in height so that should all be equal now (weird, as this was auto-exported from the icon Figma template). Replaced a few other images as well that also had incorrect dimensions.. should be good to go!

jaimecbernardo reviewed (approved):
LGTM! Thank you for the contribution!

jmaraujouy commented:
Hey, @niels9001!

I noticed that the File Locksmith icon is in RGB/A format in all sizes except for the 16x16 resolution icon, which is in 256 indexed colors. This makes the icon look "jagged" in W11 context menu when using dark mode.

![imagen](https://github.com/microsoft/PowerToys/assets/117078256/70d9f0dc-6aa3-4b47-8853-27c4a8db85e8)

If it helps, I made a small update to the icon so the 16x16 image is also in RGB/A. => ![icon-png](https://github.com/microsoft/PowerToys/assets/117078256/ecc6e974-5b6b-4e12-89da-b56995306764)

But I don't know how to upload a PR. Here's the file: [Icon.zip](https://github.com/microsoft/PowerToys/files/12185043/Icon.zip)

Thanks!
